### ISSUE TITLE
How to reduce CPU usage?

### ISSUE THREAD

**Issue by KioMioLLC** [coreybutler/fenix#99]
I am running Fenix on a very small windows 10 notebook. With nothing going on it consumes about 25% of the CPU. I have only a single server running with straight HTML. Is there any way to reduce the amount of CPU usage, so that the battery lasts longer?

**Comment by coreybutler** [comment]
That seems pretty high for CPU usage, even on a small notebook. There's nothing that's very CPU intensive in Fenix, so I'm not sure why it would be that high.

Are you using the request browser? Even that isn't very computationally expensive, but it's the only thing I can think of to turn off.

**Comment by KioMioLLC** [comment]
I am not using the request browser. The GUI only consumes about 2%. The Fenix service in resource monitor cycles about every 5 seconds between 15% and 25%. It does not matter if I stop the single server. Does that give you any clues? I've attached a screen shot with the yellow being fenix.exe

![fenix](https://user-images.githubusercontent.com/36723689/36710845-496e00f4-1b35-11e8-9c37-cab6711c2c58.png)

**Comment by coreybutler** [comment]
I think the process eating your CPU may be the underlying node-webkit engine. It's hard to tell, but the second `fenix.exe` aligns with what I usually see for the UI/server process. The behavior (or lack of change) you're describing also suggests it's the node-webkit process. If that's the case, there really isn't anything you can do about that and I won't be cutting another 2.x.x release (since v3 is imminent).

Are you running the Windows Creators Update?

**Comment by KioMioLLC** [comment]
Thank you looking into this. I am not running Windows Creators Update,
since this is a standalone device (no internet) as part of a turnkey system.

   1. Would that update help or hinder?
   2. Do you think v3 will help with this issue?

**Comment by coreybutler** [comment]
I do not think the Creators Update will help... I was actually wondering if that was the problem (since I can't replicate this and I'm _not_ on the Creators Update).

So far, v3 has shown minimal resource consumption. We built v3 from scratch on Electron (no more node-webkit), which usually doesn't have high CPU usage. Some Electron apps have very high memory usage, so we've spent alot of time to not fit into that category :)
